### PR TITLE
Add less patterns from cfgov-refresh

### PR DESCRIPTION
**Pull request by virginiacc**
Adds:
- figure img enhancements
- .superheader line height reduction
- .u-small-text small text utility
- hide/show on mobile utility classes
- percentage width utility classes
- .list with brand-color bullets

### CODE REVIEW

File: src/cf-base.less [hunk 1]
```diff
@@ -833,6 +833,66 @@ img {
     max-width: 100%;
 }
 
+/* topdoc
+  name: Figure updates
```
Comment (himedlooff) [inline src/cf-base.less]: I'd rename this to just "Figure" and pull the figure margin-left/right down in here too. I'd leave the figure margin top/bottom in that big vertical margin list.

File: src/cf-base.less [hunk 2]
```diff
@@ -833,6 +833,66 @@ img {
     max-width: 100%;
 }
```
Comment (himedlooff) [inline src/cf-base.less:830]: Very picky but we usually put two lines between topdoc comments. :stuck_out_tongue:

**Comment by himedlooff** [comment]
Everything else looks good.

File: src/cf-base.less (hunk 1 above)
Comment (virginiacc) [inline src/cf-base.less]: Good point! I've consolidated those rules.

**Comment by himedlooff** [comment]
Thanks!

**Comment by himedlooff** [comment]
Can you bump the version in bower.json and update the changelog?

**Comment by virginiacc** [comment]
Version & changelog updated. Thanks for the reminder!

**Comment by himedlooff** [comment]
Thanks, please push the new tag!

**Pull request merged**